test/e2e/framework/util.go

```diff
@@ -656,8 +652,13 @@ func podFromManifest(filename string) (*api.Pod, error) {
 
 // Run a test container to try and contact the Kubernetes api-server from a pod, wait for it
 // to flip to Ready, log its output and delete it.
-func RunKubernetesServiceTestContainer(c *client.Client, repoRoot string, ns string) {
-	path := filepath.Join(repoRoot, "test", "images", "clusterapi-tester", "pod.yaml")
+func RunKubernetesServiceTestContainer(c *client.Client, ns string) {
+	c, err := LoadClient()
```

Comment:
was this a bad rebase? `c` is passed as an argument and now you're overwriting it.